adding quota check to hortense documentation (#236)
hpcleuven/VscDocumentation: source/gent/tier1_hortense.rst

System-specific aspects
-----------------------

> Checking quota

Reviewer: `quota -> `storage quota usage` (same below)

**************

> * You can check personal and project quota by running ``my_dodrio_quota``.

Reviewer: ```
by running the ``my_dodrio_quota`` command.

> * If you want to check quota for (a) specific project(s), or project(s) that is not listed automatically, use ``-p`` option.

Reviewer 2: ```suggestion
* If you want to check quota for specific projects, or for projects that are not listed automatically, use the ``-p`` option.
```
